### DISCUSSION TITLE
How to set pdf_mode?

### DISCUSSION THREAD

**Discussion by spanhotra** [Cinnamon/kotaemon#483] [Q&A]
I want to use OCR parsing in pdf. How can I change the setting to enable that?

**Comment by taprosoft** [comment]
![image](https://github.com/user-attachments/assets/0af7492f-0d77-4c4e-87ec-22b2a46facfb)
OCR can be enabled if you choose AzureDI options. We are working on Docling to enable OCR with local models.

**Reply by vap0rtranz** [reply]
I tried the new Docling in latest build.  I noticed in logs that it was processing tables in the PDF that I uploaded.  Very nice!

BTW: Docling is CPU bound right now.  The Docling dev team is working on a v2 release to support [GPU acceleration](https://github.com/DS4SD/docling/issues/304), and fix [issues with multi-threading](https://github.com/DS4SD/docling/issues/133#issuecomment-2399681238).  So Docling loader is rather slow.  Just FYI .. I'm still excited for the future of using it :)

**Reply by dromeuf** [reply]
In your experience, is it better to use Docling for integration, or do you consider the default tools used by Kotaemon install to analyze documents and transform them into integration to be sufficient ? I'm trying to prepare a comparison and performance table (quality and accuracy for retrieving information via the prompt, not speed) on embedding models used with Kotaemon & local Ollama for local RAG, such as nomic, qwen2.5:14b, llama3.2...  By the way, has anyone ever produced such a comparative table? If you know of an article other than MTEB.
